### DISCUSSION TITLE
Looking for Contributions ?

### DISCUSSION THREAD

**Discussion by IAmRiteshKoushik** [pockethost/pockethost#247] [Q&A] [answered]
Is PocketHost looking for contributions to certain segments of the codebase / documentation ? Would be happy to jump on-board and try contributing the best of my abilities.

**Comment by benallfree** [comment, marked as answer]
Hey, thanks for the offer! Documentation is always helpful. The main area we are looking at right now is the Deno-based cloud functions. Do you have any personal use for that area of PocketHost?

**Reply by siniradam** [reply]
Hello, would it be possible to get together and have a talk? 
Meet / Discord etc? I would like to contribute as well. @benallfree I've sent an email a while ago, but didn't hear from you.

**Reply by IAmRiteshKoushik** [reply]
Yeah, I was thinking the same. If we could get together in Discord and speak a bit about the type of contributions that Pockethost is currently looking for; or the good-first issues that require addresal then we can go on and start fixing + building things.

**Reply by IAmRiteshKoushik** [reply]
I feel the README.md file can undergo a few cycles of improvement where we add in things like Table of Contents, how to Setup the project locally and other Contribution Guides.

**Reply by benallfree** [reply]
@siniradam @IAmRiteshKoushik Sure, let's try a Discord server and see how it grows https://discord.gg/pmuA3hGZ

**Reply by IAmRiteshKoushik** [reply]
Joined.
